fastplotlib/layouts/_record_mixin.py

```diff
@@ -3,12 +3,17 @@
 from multiprocessing import Queue, Process
 from time import time
 
-try:
-    import av
-except ImportError:
-    HAS_AV = False
-else:
-    HAS_AV = True
+
+def _get_av():
+    try:
+        import av
+    except ImportError:
+        raise ModuleNotFoundError(
+            "Recording to video file requires `av`:\n"
+            "https://github.com/PyAV-Org/PyAV"
+        ) from None
```

Comment:
What is the `from None`, never seen that before 😄

Reply:
This is related to exception chaining:
```py
try:
    1/0
except ZeroDivisionError:
    raise ValueError("meh")
```
Causes:
```
Traceback (most recent call last):
  File "/Users/almar/dev/py/tmp.py", line 2, in <module>
    1/0
    ~^~
ZeroDivisionError: division by zero

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/almar/dev/py/tmp.py", line 4, in <module>
    raise ValueError("meh")
ValueError: meh
```

A chain can be made explicit using the `from`  keyword:
```py
try:
    1/0
except ZeroDivisionError as err:
    raise ValueError("meh") from err
```
Then:
```
Traceback (most recent call last):
  File "/Users/almar/dev/py/tmp.py", line 2, in <module>
    1/0
    ~^~
ZeroDivisionError: division by zero

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/almar/dev/py/tmp.py", line 4, in <module>
    raise ValueError("meh") from err
ValueError: meh
```

And it can be used (as in the code this comment refers to) to tell that the exception being raised is "standalone":
```py
try:
    1/0
except ZeroDivisionError as err:
    raise ValueError("meh") from None
```

Then:
```
Traceback (most recent call last):
  File "/Users/almar/dev/py/tmp.py", line 4, in <module>
    raise ValueError("meh") from None
ValueError: meh
```

Reply:
TL;DR: The `from None` disables exception chaining.

Reply:
Ah TIL! Cleaner messages is nice :smile:

Reply:
Very cool I didn't know this was possible.